sdk/storage/azure-storage-blob/azure/storage/blob/blob_client.py

```diff
@@ -443,11 +445,8 @@ def _upload_blob_options(  # pylint:disable=too-many-statements
     def upload_blob(  # pylint: disable=too-many-locals
             self, data,  # type: Union[Iterable[AnyStr], IO[AnyStr]]
             blob_type=BlobType.BlockBlob,  # type: Union[str, BlobType]
-            overwrite=False,  # type: bool
             length=None,  # type: Optional[int]
             metadata=None,  # type: Optional[Dict[str, str]]
-            content_settings=None,  # type: Optional[ContentSettings]
-            validate_content=False,  # type: Optional[bool]
```

Comment:
should `max_concurrency` also be a `**kwarg`?

Reply:
i'm not entirely in favor of it..it's something that should be used for optimal performance,